boards/openlabs-kw41z-mini-256kib/Makefile.include

```diff
@@ -0,0 +1,2 @@
+INCLUDES += -I$(RIOTBOARD)/openlabs-kw41z-mini/include
+include $(RIOTBOARD)/openlabs-kw41z-mini/Makefile.include
```

Comment:
Are the `SLOTn_LEN` values correct for the 256Kib version?

Reply:
Good question - they are different for [openlabs-kw41z-mini](https://github.com/benemorius/RIOT/blob/openlabs/boards/openlabs-kw41z-mini/Makefile.include) and [openlabs-kw41z-mini-r1](https://github.com/benemorius/RIOT/blob/openlabs/boards/openlabs-kw41z-mini-r1/Makefile.include)